core/src/main/java/com/google/cloud/sql/core/CoreSocketFactory.java

```diff
@@ -290,7 +296,8 @@ private static SQLAdmin createAdminApiClient(HttpRequestInitializer requestIniti
     JsonFactory jsonFactory = JacksonFactory.getDefaultInstance();
     SQLAdmin.Builder adminApiBuilder =
         new Builder(httpTransport, jsonFactory, requestInitializer)
-            .setApplicationName(getApplicationName());
+            .setApplicationName(
+                String.format("%s %s", getDefaultUserAgent(), getApplicationName()));
```

Comment:
What if this is null?

Reply:
so right now, `getDefaultUserAgent` defaults to "cloud-sql-java-connector" and `getApplicationName` defaults to an empty string. So there shouldn't be any cases of either of those returning null.

Reply:
I'll probably get rid of the getDefaultUserAgent method  in the next commit though, since we don't need to set a system property. In that case I'll handle if the connector name is null.